server/src/main/java/org/opensearch/search/backpressure/stats/SearchBackpressureStats.java

```diff
@@ -46,18 +60,21 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
     public void writeTo(StreamOutput out) throws IOException {
         searchShardTaskStats.writeTo(out);
         out.writeString(mode.getName());
+        if (out.getVersion().onOrAfter(Version.V_3_0_0)) {
+            searchTaskStats.writeTo(out);
```

Comment:
That would NPE: https://github.com/opensearch-project/OpenSearch/pull/5605/files#diff-9b9d3a4d0d64b11d9291b180137965eb5dde00d0e3f437effcd88397edca60beR46

Reply:
Updated.